spec/support/capybara.rb

```diff
@@ -4,7 +4,7 @@
 
 Capybara.register_driver :headless_chrome do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: {  args: %w[no-sandbox headless disable-gpu window-size=1280,1024] }
+    "goog:chromeOptions": {  args: %w[no-sandbox headless disable-gpu window-size=1280,1024] }
```

Comment:
`Resolved issue 1786: Feature request: change chromeOptions to goog:chromeOptions [['Pri-3']]`
see: https://chromedriver.storage.googleapis.com/2.31/notes.txt